app/src/newdocs/content/marketplace.mdx

```diff
@@ -1,11 +1,15 @@
 import ScrollableAnchor from 'react-scrollable-anchor'
+import { Link } from 'react-router-dom'
 
+import links from '$shared/../links'
 import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 
 <ScrollableAnchor id="introduction-marketplace"><div>
 
 ## Introduction to the Marketplace
---content--
+The [marketplace](https://streamr.com/marketplace) is a Streamr developed application that handles the buying and selling of real-time data on the Streamr platform. The marketplace is filled with both free products and products created by data producers that wish to monetise their data. The currency of the marketplace is the <Link to={links.newdocs.dataToken}>DATA token</Link>.
+
+The marketplace is an [open source](http://github.com/streamr-dev/streamr-platform) project that works with the [Ethereum network](https://www.ethereum.org). The Ethereum network is used to coordinate access to a product's data and to provide the ledger of transactions. Products are deployed to the Ethereum network and payments are made through the [marketplace smart contract](https://etherscan.io/address/0xa10151d088f6f2705a05d6c83719e99e079a61c1). These payments are direct, buyer to seller transactions of the DATA token without a middleman fee. All of the data on the marketplace is transported via the [Streamr network](https://streamr.com/network).
```

Comment:
- Use capitalized formatting for product names (Marketplace, Core, and Network)
- Minor suggested edits for clarity

```suggestion
The Marketplace is an [open source](http://github.com/streamr-dev/streamr-platform) project that builds on the [Streamr Network](https://streamr.com/network) and the [Ethereum blockchain](https://www.ethereum.org). Ethereum is used to store a registry of data products, coordinate access to them, and to provide a secure ledger of value transactions. Products are published to the [Marketplace smart contract](https://etherscan.io/address/0xa10151d088f6f2705a05d6c83719e99e079a61c1), and the contract allows data buyers to directly transact with sellers using the DATA token, removing all traditionally present middlemen and their fees. All of the data on the Marketplace is delivered peer-to-peer via the Streamr Network.
```